libraries/core_libs/consensus/src/dag/dag_block_manager.cpp

```diff
@@ -181,17 +181,16 @@ DagBlockManager::InsertAndVerifyBlockReturnType DagBlockManager::verifyBlock(con
   }
 
   auto propose_period = db_->getProposalPeriodForDagLevel(blk.getLevel());
-  const auto expiry_period = pbft_chain_->getDagExpiryPeriod();
   // Verify DPOS
   if (!propose_period) {
     // Cannot find the proposal period in DB yet. The slow node gets an ahead block, remove from seen_blocks
     LOG(log_nf_) << "Cannot find proposal period " << *propose_period << " in DB for DAG block " << blk.getHash();
```

Comment:
can you please fix this log? if `propose_period`does not exist  `*propose_period` is UB

Reply:
Fixed